packages/posthog/src/definitions.ts

```diff
@@ -25,6 +25,14 @@ export interface PosthogPlugin {
    * @since 7.0.0
    */
   getFeatureFlag(options: GetFeatureFlagOptions): Promise<GetFeatureFlagResult>;
+  /**
+   * Get the current session ID.
+   * 
+   * Only available on Android and iOS.
+   * 
+   * @since 7.0.1
```

Comment:
```suggestion
   * @since 7.1.0
```

This should be the next version.